packages/snaps-controllers/src/snaps/endowments/enum.ts

```diff
@@ -6,4 +6,5 @@ export enum SnapEndowments {
   Cronjob = 'endowment:cronjob',
   EthereumProvider = 'endowment:ethereum-provider',
   Rpc = 'endowment:rpc',
+  WebAssemblyAccess = 'endowment:webassembly',
```

Comment:
For consistency:
```suggestion
  WebAssembly = 'endowment:webassembly',
```

Reply:
That's how I started, but `ESLint` disallows that for a good reason:
```
ESLint: 'WebAssembly' is already a global variable.(@typescript-eslint/no-shadow)
```
That's the reason why I added `Access` suffix.